packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.controller.js

```diff
@@ -0,0 +1,83 @@
+import filter from 'lodash/filter';
+import get from 'lodash/get';
+import map from 'lodash/map';
+
+export default class TelecomPackMigrationConfirmCtrl {
+  /* @ngInject */
+  constructor($q, $translate, TucPackMigrationProcess, TucToast) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.TucPackMigrationProcess = TucPackMigrationProcess;
+    this.TucToast = TucToast;
+  }
+
+  $onInit() {
+    this.process = null;
+    this.loading = {
+      migrate: false,
+    };
+    this.choosedAdditionalOptions = null;
+    this.model = {
+      acceptContracts: false,
+    };
+    this.modemTransportPrice = 9.99;
+
+    this.process = this.TucPackMigrationProcess.getMigrationProcess();
+    this.choosedAdditionalOptions = this.TucPackMigrationProcess.getOptionsSelected();
+  }
+
+  /*= ==============================
+  =            HELPERS            =
+  =============================== */
+
+  getOptionPrice(option) {
+    return this.TucPackMigrationProcess.getPriceStruct(
+      option.optionalPrice.value * option.choosedValue,
+    );
+  }
+
+  getFirstMensuality() {
+    return this.TucPackMigrationProcess.getPriceStruct(
+      this.process.selectedOffer.displayedPrice.value + 9.99,
```

Comment:
why 9.99 is hardcoded? can't you receive the price from the api?

Reply:
nope